index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Design</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+
+<body>
+    <div id="main">
+        <div id="header">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
```

Comment:
Kindly use the `header` element instead of `div` to layout your header. This helps your header to be displayed perfectly and it reduces the styles that are needed to make it perfect.